src/main/scala/reactor/core/scala/publisher/PimpMyPublisher.scala

```diff
@@ -11,17 +11,17 @@ import scala.language.implicitConversions
   */
 object PimpMyPublisher {
 
-  implicit def fluxToJFlux[T](flux: SFlux[T]): JFlux[T] = flux.asJava()
+  implicit def fluxToJFlux[T](flux: SFlux[T]): JFlux[_ <: T] = flux.asJava()
 
-  implicit def monoToJMono[T](mono: SMono[T]): JMono[T] = mono.asJava()
+  implicit def monoToJMono[T](mono: SMono[T]): JMono[_ <: T] = mono.asJava()
 
   implicit def jFluxJLong2FluxLong(jFluxLong: JFlux[JLong]): SFlux[Long] = SFlux.fromPublisher(jFluxLong.map(Long2long(_: JLong)))
 
   implicit def jFluxJInt2JFluxInt(jFluxInt: JFlux[Integer]): JFlux[Int] = jFluxInt.map[Int]((i: Integer) => Integer2int(i))
 
   implicit def jMonoJLong2JMonoLong(mono: JMono[JLong]): JMono[Long] = mono.map(Long2long(_: JLong))
 
-  implicit def jFlux2SFlux[T](jFlux: JFlux[T]): SFlux[T] = new ReactiveSFlux[T](jFlux)
-
-  implicit def jMono2SMono[T](jMono: JMono[T]): SMono[T] = new ReactiveSMono[T](jMono)
+//  implicit def jFlux2SFlux[T](jFlux: JFlux[T]): SFlux[T] = new ReactiveSFlux[T](jFlux)
+//
+//  implicit def jMono2SMono[T](jMono: JMono[T]): SMono[T] = new ReactiveSMono[T](jMono)
```

Comment:
are these no longer possible?

Reply:
Personally I don't think this is even needed anymore. But I could be wrong.

Reply:
Personally I don't think this is even needed anymore. But I could be wrong. `PimpMyPublisher` is open for public, so developer could use it. When I wrote this, it was meant for internal usage and for application to use `.asJava` and `.asScala` explicitly to make the code more readable.

Reply:
@jpork Could this commented line be removed?

Reply:
From my point of view I wouldn't provide implicit conversion between SFlux & Co and JFlux & Co. This is to dangerous and adding .asJava or .asScala as conversion code is not to much and better readable. 
Scala collections do it the same way. This conversion provoked a recursive call in SFlux which got on my nerves for quite a while. 
So I would remove the whole PimpMyPublisher thing.  But this is another story.

Reply:
> This conversion provoked a recursive call in SFlux which got on my nerves for quite a while.
> So I would remove the whole PimpMyPublisher thing. But this is another story.

Totally agree with this. Sometimes i also got recursive call to `SFlux` itself while the intention is to call `JFlux`.